web-dev/src/components/HomePage/HomePageComponent.js

```diff
@@ -1,28 +1,25 @@
 import React from "react";
 import HomeBannerComponent from "../HomeBanner/HomeBannerComponent.js";
 import NavigationTopBar from "../NavigationTopBar/index.js";
+import BoxArtList from "../BoxArtList/index.js"
 import './home-page.css';
 
-
-
 const HomePageComponent = () => {
-    // bg-color-white
     const formatted = (
         <>
-            <div className="row">
+            <div className="row bg-color">
                 <div >
                     <NavigationTopBar />
                 </div>
-                <div className="wd-add-height-to-scroll">
+                <div >
                     <HomeBannerComponent />
                 </div>
-
             </div>
 
-            <div>
-                Daily Picks
```

Comment:
Any other topic we implement will look like the "Trending Now" section. I didn't add anything else since I want to get the trending now section reviewed first. Then maybe I'll add one other section.